app/webpacker/controllers/flatpickr_controller.js

```diff
@@ -17,7 +17,7 @@ import ShortcutButtonsPlugin from "shortcut-buttons-flatpickr";
 import labelPlugin from "flatpickr/dist/plugins/labelPlugin/labelPlugin";
 
 export default class extends Flatpickr {
-  static values = { enableTime: Boolean, mode: String };
+  static values = { enableTime: Boolean, mode: String, defaultDate: String };
```

Comment:
I think it's great to indicates as a comment that `defaultDate` only takes `startOfDay` and `endOfDay` as accepted values. What do you think?

Reply:
I like the idea